vespa-hadoop/src/main/java/com/yahoo/vespa/hadoop/pig/VespaDocumentOperation.java

```diff
@@ -157,6 +160,12 @@ public String exec(Tuple tuple) throws IOException {
             if (statusReporter != null) {
                 statusReporter.incrCounter("Vespa Document Operation Counters", "Document operation failed", 1);
             }
+            if (printIdOnError) {
+                Schema inputSchema = getInputSchema();
+                Map<String, Object> fields = TupleTools.tupleMap(inputSchema, tuple);
+                String docId = TupleTools.toString(fields, template);
+                System.out.println("Error occur when processing document with docID: " +docId);
```

Comment:
This should probably not be a system out here. Use the warnlog function. But, isn't this information already in the line below as the entire tuple is output to warnlog? If so, I don't see the point of adding additional output here. If not, rather change the exception out below to explicitly add the docid. No configuration should be necessary for this.

Reply:
Make sense. BTW, if we want to always print the docID out (System.out.println() will be at line 151, before the create function is called). Is it acceptable that we just print it out or should we make it a property just like in this PR? Thanks!

Reply:
@lesters

Reply:
Well, we definitely shouldn't be stdout'ing the document id for each document by default, as grid feeds usually have quite a large number of documents. If you want such a feature, add a `verbose` or `debug` property, and if this property is set you can output it. Make sure this property is by default off.